genomicFileHandler/genomic_file_handlers.py

```diff
@@ -243,87 +223,87 @@ def open_bam_file(file_name):
 def ascii2phred33(x):
     '''Put in an ASCII string, return a Phred+33 score.'''
     return ord(x)-33
-    
-    
+
+
 def phred33toascii(x):
     '''Put in a Phred33 score, return the character.'''
     return chr(x+33)
-    
+
 
 def p2phred(p, max_phred=inf):
     '''Convert p-value to Phred-scale quality score.'''
-    
+
     if p == 0:
         Q = max_phred
-        
+
     elif p == 1:
         Q = 0
-        
+
     elif p<0 or p>1:
         Q = nan
-    
-    elif p > 0:    
+
+    elif p > 0:
         Q = -10 * math.log10(p)
         if Q > max_phred:
             Q = max_phred
-            
+
     elif math.isnan(p):
         Q = nan
-                
+
     return Q
-    
-    
-    
+
+
+
 def phred2p(phred):
     '''Convert Phred-scale quality score to p-value.'''
     return 10**(-phred/10)
-    
+
 
 def findall_index(mylist, tolookfor):
     '''Find all instances in a list that matches exactly thestring.'''
     all_indices = [i for i,mylist in enumerate(mylist) if mylist == tolookfor]
     return all_indices
-    
-    
+
+
 def findall_index_regex(mylist, pattern):
     '''Find all instances in a list that matches a regex pattern.'''
     all_indices = [i for i,mylist in enumerate(mylist) if re.search(pattern, mylist)]
```

Comment:
`i,mylist in enumerate(mylist)` can be rewritten as `i, item in enumerate(mylist)` to avoid confusion.